### PR TITLE
dev-util/pwndbg: update dependencies

### PR DESCRIPTION
**Pull request by hamarituc**
New packages are needed to fulfill dependencies:

 * `dev-python/colored-traceback`
 * `dev-utils/pwntools`

### CODE REVIEW

File: dev-python/colored-traceback/colored-traceback-0.3.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+DESCRIPTION="Automatically color Python's uncaught exception tracebacks"
+HOMEPAGE="https://github.com/staticshock/colored-traceback.py"
+SRC_URI="mirror://pypi/${P:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="ISC"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="dev-python/pygments[${PYTHON_USEDEP}]"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
```
Comment (thesamesam) [inline dev-python/colored-traceback/colored-traceback-0.3.0.ebuild:20]: Do we need this?

File: dev-util/pwntools/metadata.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+	<maintainer type="person" proxied="yes">
+		<email>mario.haustein@hrz.tu-chemnitz.de</email>
+		<name>Mario Haustein</name>
+	</maintainer>
+	<maintainer type="project" proxied="proxy">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
+	<upstream>
+		<remote-id type="github">Gallopsled/pwntools</remote-id>
```
Comment (thesamesam) [inline dev-util/pwntools/metadata.xml:13]: pypi remote-id

File: dev-util/pwntools/pwntools-4.10.0_beta0.ebuild [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+PATCHES=(
+	"${FILESDIR}/${PN}-4.0.0_do_not_mess_with_docs.patch"
+	"${FILESDIR}/${PN}-4.1.0_update_deps.patch"
+)
+
+src_install() {
+	distutils-r1_src_install
+
+	rm -r "${D}/usr/pwntools-doc" || die
```
Comment (thesamesam) [inline dev-util/pwntools/pwntools-4.10.0_beta0.ebuild:55]: ${ED}

File: dev-python/colored-traceback/colored-traceback-0.3.0.ebuild (hunk 1 above)
Comment (hamarituc) [reply dev-python/colored-traceback/colored-traceback-0.3.0.ebuild]: fixed

File: dev-util/pwntools/metadata.xml (hunk 2 above)
Comment (hamarituc) [reply dev-util/pwntools/metadata.xml:13]: fixed

File: dev-util/pwntools/pwntools-4.10.0_beta0.ebuild (hunk 3 above)
Comment (hamarituc) [reply dev-util/pwntools/pwntools-4.10.0_beta0.ebuild]: fixed

**Comment by thesamesam** [comment]
Thank you!

**Pull request closed without merging**